.changeset/selfish-grapes-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"hue-map": minor
```

Comment:
Should be a major change, this will break existing code

Reply:
I know its a breaking change but I wasn't sure what the semantic versioning approach was wrt the package not having a 1.0 release. Do we just jump to 1.0 or does the minor release before 1.0 act as a breaking change?

Reply:
True technically it does, but I reckon this package is good to release at 1.0.0 😎